### DISCUSSION TITLE
Find a specific sequence in 10x multiome

### DISCUSSION THREAD

**Discussion by ctmoraes** [satijalab/seurat#6043] [General]
This can’t be too hard, but I do not know how to do it. I am analyzing scRNA and scATAC in a mouse sample with a conditional deletion of part of a gene. The gene is still detected as only the last exon is deleted. I want to search for that specific region to find my knockout cells. It got to be similar to a motif search, but for only one specific sequence.  Please help! Thx!

**Comment by timoast** [comment]
I would try adding that particular exon that's deleted as a different gene in the gtf file you're using for quantification, then you'll be able to get counts for only that exon and try to work out from that which cells have a deletion and which don't.
